crates/astria-composer/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "astria-composer"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-trait = { workspace = true }
+color-eyre = { workspace = true }
+figment = { workspace = true, features = ["toml", "env"] }
+tokio = { workspace = true }
+serde = { workspace = true, features = ["derive"]  }
+clap = { workspace = true, features = ["derive"] }
+thiserror = { workspace = true }
+tracing = { workspace = true }
+tracing-subscriber = { workspace = true, features = ["ansi", "json"] }
+reqwest = { workspace = true, features = ["json"] }
+# TODO: should these be moved to workspace? 
```

Comment:
Yeah, should be done. When adding dependencies to a project it makes sense to check if any of the other crates have that dependency.

Reply:
should that be in this pr or a different one? would change lines in `conductor` and `sequencer-relayer` as well

Reply:
This is the appropriate place to make the change.